model-server/build.gradle.kts

```diff
@@ -117,8 +117,15 @@ val cucumber = task("cucumber") {
     }
 }
 
+tasks.register<Copy>("copyApis") {
+    from("../api/")
+    include("*.yaml")
+    into(project.layout.projectDirectory.dir("api"))
```

Comment:
Why not putting this into the resource folder?

Reply:
makes more sense and also reduces complexity of the dockefile